fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -2248,6 +2252,44 @@ public ParseNode visitShowProcesslistStatement(StarRocksParser.ShowProcesslistSt
         return new ShowProcesslistStmt(isShowFull);
     }
 
+    @Override
+    public ParseNode visitRestoreStatement(StarRocksParser.RestoreStatementContext context) {
+        QualifiedName qualifiedName = getQualifiedName(context.qualifiedName());
+        LabelName labelName = qualifiedNameToLabelName(qualifiedName);
+        StarRocksParser.IdentifierContext repo = context.identifier();
+
+        List<TableRef> tblRefs = new ArrayList<>();
+        for (StarRocksParser.RestoreTableDescContext tableDescContext : context.restoreTableDesc()) {
+            StarRocksParser.QualifiedNameContext qualifiedNameContext = tableDescContext.qualifiedName();
+            qualifiedName = getQualifiedName(qualifiedNameContext);
+            TableName tableName = qualifiedNameToTableName(qualifiedName);
+            PartitionNames partitionNames = null;
+            if (tableDescContext.partitionNames() != null) {
+                partitionNames = (PartitionNames) visit(tableDescContext.partitionNames());
+            }
+            StarRocksParser.IdentifierContext alias = tableDescContext.identifier();
+            TableRef tableRef =
+                    new TableRef(tableName, Optional.ofNullable(alias).map(RuleContext::getText).orElse(null),
```

Comment:
((Identifier) visit(context.identifier())).getValue()

Reply:
Reading the code, I understand the use "((Identifier) visit(context.identifier())).getValue()" can replace '`' , Thank you.